native/cocos/bindings/jswrapper/v8/ScriptEngine.cpp

```diff
@@ -509,14 +509,8 @@ ScriptEngine::ScriptEngine()
     #endif
 }
 
-ScriptEngine::~ScriptEngine() {
-    #if !CC_EDITOR
-    if (gSharedV8) {
-        delete gSharedV8;
-        gSharedV8 = nullptr;
```

Comment:
Should only don't delete it on Android. And please add some comment about the reason.